### DISCUSSION TITLE
Welcome to surgical_robotics_challenge Discussions!

### DISCUSSION THREAD

**Discussion by adnanmunawar** [surgical-robotics-ai/surgical_robotics_challenge#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rubenray** [comment]
Hi. we are installing the challenge files, I was wondering if someone could help. we having this issue:

when we try to star the simulator:

ruben@ruben-500-505la:~/ambf/bin/lin-x86_64$ ./ambf_simulator --launch_file surgical_robotics_challenge/launch.yaml -l 0,1,3,4,14,15 -p 120 -t 1 --override_max_comm_freq 120

then

Failed to Load surgical_robotics_challenge/launch.yaml
ERROR! COULDN'T DETERMINE THE CORRECT ADF LOADER FOR THE FILE "surgical_robotics_challenge/launch.yaml"

I was wondering if someone have any idea about that issue  thanks for your support

**Reply by adnanmunawar** [reply]
Hello, I think the path to the ‘launch.yaml’ file may not be correct. If the ‘surgical_robotics_challenge’ folder is located in your home folder then please add the ‘-/‘ before it, e.g. ‘ ./ambf_simulator --launch_file ~/surgical_robotics_challenge/launch.yaml -l 0,1,3,4,14,15 -p 120 -t 1 --override_max_comm_freq 120’. If it’s in a different folder just set it accordingly.
